starters/gatsby-wordpress-starter/src/templates/pagesIndex.jsx

```diff
@@ -24,3 +43,24 @@ export default PageIndexTemplate
 export function Head() {
 	return <Seo title="All pages" />
 }
+
+export const pageQuery = graphql`
```

Comment:
Since we are fetching all the pages at build time in `gatsby-node.js`, I don't think we require a page query here.